file_transfer_protocol/server.py

```diff
@@ -0,0 +1,34 @@
+import socket                   # Import socket module
+
+ONE_CONNECTION_ONLY = True      # Set this to False if you wish to continuously accept connections
+
+port = 12312                    # Reserve a port for your service.
+s = socket.socket()             # Create a socket object
+host = socket.gethostname()     # Get local machine name
+s.bind((host, port))            # Bind to the port
+s.listen(5)                     # Now wait for client connection.
+
+print('Server listening....')
+
+while True:
+    conn, addr = s.accept()     # Establish connection with client.
+    print('Got connection from', addr)
+    data = conn.recv(1024)
+    print('Server received', repr(data))
+
+    filename='mytext.txt'
+    f = open(filename,'rb')
+    l = f.read(1024)
+    while (l):
```

Comment:
L is a VERY ambiguous identifier.  Is it a one or an ell.  Please avoid single character variable names like the plague that they are.